core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -177,6 +192,10 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
           <div class="additional-metrics collapsed">
             <ul>
               <li>
+                <strong>Locality Level Summary: </strong>
```

Comment:
What do you think about a new table "Locality Levels Summary" displayed by a checkbox in additional metric ?